App.js

```diff
@@ -1,38 +1,61 @@
 import React, { Component } from 'react';
 import { StyleSheet, Text, View, KeyboardAvoidingView, Button, TextInput } from 'react-native';
 import MyHeader from 'ToDoProject/Components/MyHeader'
-import MainBackground from 'ToDoProject/Components/MainBackground'
-import InputBottom from 'ToDoProject/Components/inputPart'
-import { Header } from 'react-native-elements';
+import TaskList from 'ToDoProject/Components/TaskList'
+import InputNote from 'ToDoProject/Components/InputNote'
 export default class MainPage extends Component {
   constructor(props) {
     super(props);
-    this.state = { text: '' };
+    this.id = 1;
+    this.state = {
+
+      notes: [
+        { key: '1', description: 'task1', isDone: false },
+
+      ]
+    };
   }
 
+  addNote = (text) => {
+    if (text != '') {
+      this.id++;
+      this.state.notes.push({
```

Comment:
don't modify state directly
create new array 
`let notes = [this.state.notes, ... newNote];
this.setState ({notes});`